core/src/main/java/org/apache/calcite/sql/SqlKind.java

```diff
@@ -966,6 +966,9 @@
   /** The {@code FUSION} aggregate function. */
   FUSION,
 
+  /** The {@code INTERSECTION} aggregate function. */
+  INTERSECTION,
```

Comment:
Please also add `INTERSECTION` in `AGGREGATE` EnumSet [1].

https://github.com/apache/calcite/blob/4c3cef95377ea16efca037012385c40822b4426e/core/src/main/java/org/apache/calcite/sql/SqlKind.java#L1159

Reply:
@DonnyZone  Thank you for the suggestion. I added INTESECTION, to the Aggregate enumset. I also noticed that ANY_VALUE is also not added so added it.

Reply:
That's great! Thanks;)